CHANGES.rst

```diff
@@ -4,6 +4,14 @@ Change history
 2.0a2 (unreleased)
 ------------------
 
+- Replace manual comparisons of buildout options to ``False`` with a
+  ``bool_option`` method.
+  [saily]
+
+- Removed some plugins and replaced them with ``flake8`` plugins. Please
```

Comment:
@saily I know, I would just prefer to be a bit more explicit and name the options that have been removed.